dev-docs/modules/bidstailamedia.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: bidder
+title: stailamedia Bidder
```

Comment:
should be bidstail

Reply:
Stailamedia is correct: https://www.stailamedia.com/ 

Sales House from switzerland 😁